code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -11,34 +11,45 @@
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	pixel_y = 22
 	appearance_flags = KEEP_TOGETHER
+	var/mob/living/occupant
```

Comment:
Could you document this variable and make sure to clean it up on `Destroy()` to prevent hard dels?

Reply:
`machinery` already calls `set_occupant(null)` on del so `Destroy()` shouldn't ever reach with a non null occupant